targets/TARGET_NUVOTON/nu_bitutil.h

```diff
@@ -17,6 +17,9 @@
 #ifndef NU_BIT_UTIL_H
 #define NU_BIT_UTIL_H
 
+#if defined(TOOLCHAIN_IAR) && defined(TARGET_M0)
```

Comment:
It would be better to use mbed agnostic to find out toolchain being used, instead of ``TOOLCHAIN_`` use toolchain specific symbol, for IAR it shall be `__ICCARM__`